scrapy/downloadermiddlewares/retry.py

```diff
@@ -70,8 +72,10 @@ def _retry(self, request, reason, spider):
             retryreq.meta['retry_times'] = retries
             retryreq.dont_filter = True
             retryreq.priority = request.priority + self.priority_adjust
+            self.stats.inc_value('retry/count')
```

Comment:
Yeah, this is true. My point is that you can spot it without adding this information to stats: status codes and exceptions to retry are static, they don't depend on website - you can check settings and EXCEPTIONS_TO_RETRY list. You check `retry/count`, notice that the value is high (=> there is some issue with crawling this website), and then investigate it further: check response status codes and downloader exceptions. So this PR does this minimal change which allows to spot and debug the problem.

I'm not opposed to adding more information to stats, it is just a bit more controversial IMHO (https://github.com/scrapy/scrapy/issues/2173 becomes more verbose). But more stats could make it easier to debug retry issues, that's true.